pkg/controller.v1/mpi/mpijob_controller_test.go

```diff
@@ -709,6 +709,93 @@ var _ = Describe("MPIJob controller", func() {
 		})
 	})
 
+	Context("Test launcher's Intel MPI handling", func() {
+		It("Should create a launcher job with Intel MPI env variables", func() {
+			getIMPIValues := func(envs []corev1.EnvVar) (bootstrap, exec string) {
+				for _, env := range envs {
+					if env.Name == "I_MPI_HYDRA_BOOTSTRAP" {
+						bootstrap = env.Value
+					}
+					if env.Name == "I_MPI_HYDRA_BOOTSTRAP_EXEC" {
+						exec = env.Value
+					}
+				}
+
+				return bootstrap, exec
+			}
+
+			By("By creating MPIJobs with and without preset env variables")
+
+			testCases := map[string]struct {
+				envVariables  map[string]string
+				expectedAdded int
```

Comment:
```suggestion
			testCases := map[string]struct {
				envVariables  map[string]string
				expectedIntelMPIenvVariables  map[string]string
```

Hiding expected env vars would miss the bugs.